### ISSUE TITLE
Task View in Windows 10

### ISSUE THREAD

**Issue by pythoninthegrass** [rbreaves/kinto#434]
Thanks for making this! Excited to use it in Windows 10 (21318.1000) to mimic my Mac setup.

Trying to add [Task View](https://www.pcworld.com/article/2952864/how-to-use-windows-10s-task-view-and-virtual-desktops.html) as an option under `Basic Remap` (lines 270-272) section and it's not working:
```
; Mission Control/Expose
;$LCtrl up:: Send #{Tab}             ; MacModifiers
;MButton::Send #{Tab}                ; MacModifiers
```
Whereas under the same area (lines 262-268), was able to remap media keys successfully:
```
; Media Keys for Logitech Apple keyboard
;F8::Media_Prev                      ; MacModifiers
;F10::Media_Play_Pause               ; MacModifiers
;F9::Media_Next                      ; MacModifiers
;F11::Volume_Mute                    ; MacModifiers
;F12::Volume_Down                    ; MacModifiers
;F13::Volume_Up                      ; MacModifiers
```
If I'm interpreting how `kinto.ahk` gets called correctly, it appears that perl uncomments the selected keyboard modifiers through regex? Doesn't seem to matter for my basic code blocks as the media keys will work regardless of comments.

**Comment by rbreaves** [comment]
Yes on the Perl & you only need to add the Mac/win/chrome/ibm comments if you really need something to toggle w/ kb type & your remap for task view is a little strange but understandably hard perhaps to work out due to conflict reason w/ system os hotkeys it seems.

Macs use a media keys for expose afaik so to remap that to expose it probably ought to be Send #{Tab} like you have but not on Ctrl up. That physical still gets used & I think I already have an action happening on Ctrl up.

Also your physical Ctrl key is a Win key typically under Kinto. The Tab switcher stuff tends to cut through the remap however because it needs to to properly handle the actions that ought to occur.

**Comment by pythoninthegrass** [comment]
[Macs use both](https://support.apple.com/guide/mac-help/open-windows-spaces-mission-control-mh35798/mac) the media key (F4 on my Logitech Apple keyboard) and ctrl-up.

I'd be happy with either if there's a way to map them.  Trying to comment out the `Hack to disable start menu on winkey` allows for using the Apple mapping for Task View, but breaks Del (Backspace.)
```
    ; Hack to disable start menu on winkey
    ; Static - Does not apply to IBM or Chromebooks
;   ; $LCtrl up::Send {Ctrl down}{LWin up}{Ctrl up}            ; Default
```

**Comment by rbreaves** [comment]
I think you are misunderstanding "$LCtrl up" - that is referring to Left Control key triggering on keypress up - not a combo w/ the up arrow key!

If you want/need to remap Physical Control key + Up Arrow then that ought to be very simple #Up:: Send #{Tab}.

If #{Tab} fails then try {LWin down}{Tab}{LWin up}.

Will leave this open for now for inclusion later.

**Comment by n0irx** [comment]
If you just want to open the task view, just add: 

```
!Tab::Send {LWin down} + {Tab};{LWin up}
```

It can't cycle the desktops, though.
